crates/optimism/evm/src/config.rs

```diff
@@ -17,6 +17,9 @@ pub fn revm_spec_by_timestamp_after_bedrock(
     chain_spec: impl OpHardforks,
     timestamp: u64,
 ) -> OpSpecId {
+    // if chain_spec.is_interop_active_at_timestamp(timestamp) {
+    //     OpSpecId::INTEROP
+    // } else
```

Comment:
afaik there are no evm changes for interop?
possible that we don't need to roll a new spec but probably should anyway for consistency 
FYI @klkvr